airrohr-firmware/airrohr-firmware.ino

```diff
@@ -3935,8 +3936,17 @@ static void powerOnTestSensors() {
 	if (cfg::htu21d_read) {
 		debug_outln_info(F("Read HTU21D..."));
 		if (!htu21d.begin()) {
-			debug_outln_error(F("Check HTU21D wiring"));
-			htu21d_init_failed = true;
+      debug_outln_info(F("Read Si7021 replacement for HTU21D ..."));
+      Wire.beginTransmission(HTU21DF_I2CADDR);
+      Wire.write(HTU21DF_READREG);
+      Wire.endTransmission();
+      Wire.requestFrom(HTU21DF_I2CADDR, 1);
```

Comment:
so this seems to be a ugly workaround for 
https://github.com/adafruit/Adafruit_HTU21DF_Library/blob/master/Adafruit_HTU21DF.cpp#L49

wouldn't it be better to submit this as a PR to that library? then this code needs no modification.